Update cross-tenant-mailbox-migration.md (#10986)
MicrosoftDocs/microsoft-365-docs: microsoft-365/enterprise/cross-tenant-mailbox-migration.md

## Licensing

> Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (one-time fee) and can be assigned either on the source or target user object. This license also covers OneDrive for Business migration. Please contact your Microsoft account team for details.

Reviewer: Update this "This license also covers OneDrive for Business migration link to content" to add the link to the documentation for this https://learn.microsoft.com/microsoft-365/enterprise/cross-tenant-onedrive-migration

Author: done

> Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (onetime fee) and can be assigned either on the source or target user object. This license also covers the OneDrive for Business migration. Please contact your Microsoft account team for details.

Reviewer 2: ```suggestion
Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (one-time fee) and can be assigned either on the source or target user object. This license also covers OneDrive for Business migration. Please contact your Microsoft account team for details.
```

## Licensing

> Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (one-time fee) and can be assigned either on the source or target user object. This license also covers [OneDrive for Business migration](https://learn.microsoft.com/microsoft-365/enterprise/cross-tenant-onedrive-migration). Please contact your Microsoft account team for details.

Reviewer 2: ```suggestion
Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (one-time fee) and can be assigned either on the source or target user object. This license also covers [OneDrive for Business migration](/microsoft-365/enterprise/cross-tenant-onedrive-migration). Contact your Microsoft account team for details.
```

While Teams meetings are moved, the meeting URL isn't updated when items migrate cross-tenant. Since the URL will be invalid in the target tenant, you must remove and recreate Teams meetings.

### What content is migrated cross-tenant?

> When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions and Purges is migrated.  

Reviewer 2: ```suggestion
When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions, and Purges is migrated.  
```

> When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions, and Purges is migrated.  

Reviewer 2: ```suggestion
When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions, and Purges are migrated.  
```

### Does the Teams chat folder content migrate cross-tenant?

No, the Teams chat folder content does not migrate cross-tenant. However, once the mailbox has been migrated cross-tenant, the Teams chat folder content will be available for source tenant admins to search and export using a content search.

> ### What content is migrated cross-tenant then?

Reviewer: Let's remove the then and move this above the chat folder and remove the Then.

Author: done

> When a mailbox is migrated cross-tenant with this feature, only user visible content in the mailbox, also known as Top Of Information Store (email, contacts, calendar, tasks, and notes) and recoverable items subfolders except SubstrateHolds is migrated.  

Reviewer 2: ```suggestion
When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and recoverable items subfolders except SubstrateHolds are migrated.  
```

> When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and recoverable items subfolders except SubstrateHolds are migrated.  

Reviewer: Update this line: 
When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions and Purges.

Author: Updated to: 
"When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions and Purges is migrated. "

Although I still think it's worth mentioning SubstrateHolds is dropped from there.

- **Issue: Post migration Teams functionality in the source tenant will be limited.** After the mailbox is migrated to the target tenant, Teams in the source tenant will no longer have access to the user's mailbox. So, if a user logs into Teams with the source tenant credential, then there will be a loss of functionality such as the inability to update your profile picture, no calendar application, and an inability to search and join public teams.

> - **Issue: Auto Expanded archives cannot be migrated.** The cross-tenant migration feature supports migrations of the primary mailbox and archive mailbox for a specific user. If the user in the source however has an auto expanded archive enabled – meaning more than one archive mailbox (aux archives), the feature is able to migrate them as long as there's no more than 10 aux archives, if there's more than 10 aux archives the migration will fail.

Reviewer 2: ```suggestion
- **Issue: Auto Expanded archives cannot be migrated.** The cross-tenant migration feature supports migrations of the primary mailbox and archive mailbox for a specific user. If the user in the source however has an auto expanded archive enabled – meaning more than one archive mailbox (aux archives), the feature is able to migrate them as long as there are no more than 10 aux archives. If there are more than 10 aux archives, the migration will fail.
```

> - **Issue: Auto Expanded archives cannot be migrated.** The cross-tenant migration feature supports migrations of the primary mailbox and archive mailbox for a specific user. If the user in the source however has an auto expanded archive enabled – meaning more than one archive mailbox (aux archives), the feature is able to migrate them as long as there are no more than 10 aux archives. If there are more than 10 aux archives, the migration will fail.

Reviewer: We already have this posted in this document under Do auto-expanded archive mailboxes move?
Yes, if the user in source has auto-expanding archives enabled and has additional auxiliary archives, cross-tenant mailbox migration will work. We support moving users that have no more than 12 auxiliary archive mailboxes.

So we just need to remove this line completly

Author: done
